src/options.rs

```diff
@@ -545,6 +571,11 @@ where
             Arg::new("vtable-generation")
                 .long("vtable-generation")
                 .help("Enables generation of vtable functions."),
+            Arg::new("remove-function-prefix")
+                .long("remove-function-prefix")
+                .multiple_occurrences(true)
+                .takes_value(true)
+                .help("Remove prefix when generating Rust function name.")
```

Comment:
Adding a new flag that's only useful to test an implementation detail is not great. Let's test this in `bindgen-integration` instead?

Reply:
I don't feel strongly about this being a flag, but I do think this could be useful for C projects that want Rust bindings that expose function names that only differ from their associated link names by the removal of a common prefix.